test/testutils.js

```diff
@@ -100,27 +100,34 @@ testUtils.checkSetup = function (content, options, target) {
  * Create check arguments with Shadow DOM. Target can be inside or outside of Shadow DOM, queried by
  * adding `id="target"` to a fragment. Or specify a custom selector as the `targetSelector` argument.
  *
- * @param Node|String 	Stuff to go into the fixture (html or node)
+ * @param Node|String 	Stuff to go into the fixture (html string or DOM Node)
  * @param Node|String 	Stuff to go into the shadow boundary (html or node)
- * @param String  			Target selector for the check, can be inside or outside of Shadow DOM (default: '#target')
  * @param Object  			Options argument for the check (optional, default: {})
+ * @param String  			Target selector for the check, can be inside or outside of Shadow DOM (optional, default: '#target')
  * @return Array
  */
-testUtils.shadowCheckSetup = function (content, shadowContent, targetSelector, options) {
+testUtils.shadowCheckSetup = function (content, shadowContent, options, targetSelector) {
```

Comment:
Why the switch?

Reply:
To make it more consistent with `checkSetup`. There are no instances of `targetSelector`, yet there are options passed in. I wanted to preserve the functionality but make the two methods more consistent